dialog/en-us/installed.dialog

```diff
@@ -0,0 +1,2 @@
+yes, {skill} is installed
+yes, {skill}'s installed
```

Comment:
Let's leave out the second version.  For a skill like "NPR News" it would be "yes, NPR news's installed".  The first one should work all the time.

Reply:
For me, "NPR news's installed" still sounds natural, possibly even more natural in casual speech. I'm not really familiar with with the way mycroft speaks in general though so I'll happily change this :)